### PR TITLE
adds google-java-format plugin to gradle.

### PR DESCRIPTION
**Pull request by rkirov**
Document how to reformat the sourcecode and run codebase through `gradle
googleJavaFormat`.

### CODE REVIEW

File: src/main/java/com/google/javascript/gents/CollectModuleMetadata.java [hunk 1]
```diff
@@ -172,12 +182,8 @@ void registerProvidesModule(Node n, String file, String namespace) {
      * are considered exported from the file, but not directly provided. This is to determine what
      * namespaces other files are allowed to reference with 'goog.require'.
      *
-     * For example,
-     * goog.module('A.B');
-     * exports = ...;
-     * exports.C = ...;
-     * exports.C.D = ...;
-     * Would result in providesObjectChildren['A.B'] = {'C'}
+     * <p>For example, goog.module('A.B'); exports = ...; exports.C = ...; exports.C.D = ...; Would
```
Comment (dpurp) [inline src/main/java/com/google/javascript/gents/CollectModuleMetadata.java]: looks like this is missing [pre] tags

File: src/main/java/com/google/javascript/gents/ModuleConversionPass.java [hunk 2]
```diff
@@ -277,11 +278,9 @@ public boolean shouldTraverse(NodeTraversal t, Node n, Node parent) {
   /**
    * Converts a Closure goog.require call into a TypeScript import statement.
    *
-   * The resulting node is dependent on the exports by the module being imported:
-   * import localName from "goog:old.namespace.syntax";
-   * import {A as localName} from "./valueExports";
-   * import * as localName from "./objectExports";
-   * import "./sideEffectsOnly"
+   * <p>The resulting node is dependent on the exports by the module being imported: import
```
Comment (dpurp) [inline src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: here too

File: src/main/java/com/google/javascript/clutz/Depgraph.java [hunk 3]
```diff
@@ -72,12 +72,17 @@ static Depgraph parseFrom(List<String> fileNames) {
       try {
         String depgraph = Files.toString(new File(depgraphName), UTF_8);
         List<List<?>> list =
-            new Gson().fromJson(depgraph, new TypeToken<List<List<?>>>() { /* empty */ }.getType());
+            new Gson()
+                .fromJson(
+                    depgraph,
+                    new TypeToken<List<List<?>>>() {
+                      /* empty */
```
Comment (mprobst) [inline src/main/java/com/google/javascript/clutz/Depgraph.java]: Maybe you could remove the comment to get a saner formatting?

Comment (rkirov) [inline src/main/java/com/google/javascript/clutz/Depgraph.java]: yep

File: src/main/java/com/google/javascript/gents/CollectModuleMetadata.java (hunk 1 above)
Comment (rkirov) [inline src/main/java/com/google/javascript/gents/CollectModuleMetadata.java]: added

File: src/main/java/com/google/javascript/gents/ModuleConversionPass.java (hunk 2 above)
Comment (rkirov) [inline src/main/java/com/google/javascript/gents/ModuleConversionPass.java]: done

**Comment by rkirov** [comment]
merged

**Pull request closed without merging**